metadata/moa/scripts/nbconverted/2.map-broad_id.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import rdkit.Chem.inchi as inchi
+
+
+# Sample files from each version of the repurposing hub are read
+
+# In[2]:
+
+
+samples_2017 = pd.read_csv('clue/repurposing_samples_20170327.txt',usecols=['broad_id','pert_iname','InChIKey'],delimiter='\t', comment='!', encoding='iso-8859-1')
```

Comment:
the `comment='!'` is pretty neat... I have been using [`skiprows=9`](https://github.com/broadinstitute/lincs-cell-painting/blob/master/metadata/moa/scripts/nbconverted/0.merge-repurposing-compounds.py#L32) but defining the comment I think is more elegant